app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controllers/gamesController");
+
+app.get("/api", (req, res, next) => {
+  res.status(200).send({ msg: "Connected" });
```

Comment:
You don't need this endpoint, avoid writing code that isn't needed by your tickets. Stay on target!